src/main/kotlin/app/revanced/extensions/Extensions.kt

```diff
@@ -21,6 +25,24 @@ import org.w3c.dom.Node
  */
 internal fun MethodFingerprint.toErrorResult() = PatchResultError("Failed to resolve $name")
 
+fun MethodFingerprint.resolveMany(context: BytecodeContext, classes: Iterable<ClassDef>) = sequence {
+    for (classDef in classes) // search through all classes for the fingerprint
+        yieldAll(this@resolveMany.resolveMany(context, classDef))
+
+}
+/**
+ * Resolve a [MethodFingerprint] against all methods of a [ClassDef].
+ *
+ * @param forClass The class on which to resolve the [MethodFingerprint] in.
+ * @param context The [BytecodeContext] to host proxies.
+ * @return A sequence of [MethodFingerprintResult].
+ */
+fun MethodFingerprint.resolveMany(context: BytecodeContext, forClass: ClassDef) = sequence {
```

Comment:
This extension function already exists inside the `extensions` package from ReVanced Patcher. You can resolve a fingerprint given a `ClassDef.` Consider removing this addition and use the existing function.

Reply:
I did not see that when I looked but I'll definitely have another look.